tests/atest/transformers/RenameKeywords/expected/with_library_name.robot

```diff
@@ -0,0 +1,6 @@
+*** Keywords ***
+Library Name
+    Process.Only Keyword Name
+    RPA.Browser.playwright.Take Screenshot
```

Comment:
Should be     RPA.Browser.Playwright.Take Screenshot

Reply:
Actually library name was skipped so it was expected behaviour. But I've updated the code and now library name is also capitalized.